app/main.py

```diff
@@ -1 +1,45 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
You have an extra `else` statement in your code (checklist point 1). It's better to avoid unnecessary `else` statements. The function will return `self.appetite` if `self.is_hungry` is `True`, and if not it will get to the line `return 0` anyway. So you can safely remove the `else` statement.